content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -106,7 +106,8 @@ different Kubernetes components.
 | `EndpointSliceTerminatingCondition` | `false` | Alpha | 1.20 | 1.21 |
 | `EndpointSliceTerminatingCondition` | `true` | Beta | 1.22 | |
 | `EphemeralContainers` | `false` | Alpha | 1.16 | 1.22 |
-| `EphemeralContainers` | `true` | Beta | 1.23 | |
+| `EphemeralContainers` | `true` | Beta | 1.23 | 1.24 |
+| `EphemeralContainers` | `true` | GA | 1.25 | - |
```

Comment:
There's a different table for graduated features; make sure to move this there.

Reply:
:+1:  thanks!